rust/src/measurements/make_private_lazyframe/mod.rs

```diff
@@ -110,7 +116,6 @@ where
         }
 
         match &self {
-            #[cfg(feature = "contrib")]
```

Comment:
Is this correct? I'm surprised that `LogicalPlan::Projection` is contrib, but it's removed here.

Reply:
Whoops, thanks.